lib/src/git.rs

```diff
@@ -213,28 +213,54 @@ pub enum GitExportError {
     InternalGitError(#[from] git2::Error),
 }
 
-/// Reflect changes made in the Jujutsu repo compared to our current view of the
-/// Git repo in `mut_repo.view().git_refs()`. Returns a list of names of
-/// branches that failed to export.
-// TODO: Also indicate why we failed to export these branches
+/// Export changes to branches made in the Jujutsu repo compared to the Git
+/// repo. Returns a list of names of branches that failed to export.
```

Comment:
Might be good to say "... compared to our last seen view of the Git repo according to  .jj/repo/git_last_seen_refs" or something like that to be clear, even though there's some more detail.

Reply:
I edited this a bit (will upload the result shortly once I resolve conflicts). I think the additional details around the `with_last_seen_refs` function in the following commit cover the rest, but leave another comment there if not.